backend/scripts/development-seed.ts

```diff
@@ -0,0 +1,63 @@
+import { NestFactory } from '@nestjs/core'
+import { AppModule } from 'src/app.module'
+import { UserService } from 'src/user/user.service'
+import { LocationService } from '../src/location/location.service'
+
+const create = async () => {
+  const app = await NestFactory.create(AppModule)
+  const locationService = app.get(LocationService)
+  const userService = app.get(UserService)
+
+  const l1 = await locationService.create()
+  const l2 = await locationService.create()
+  const l3 = await locationService.create()
+
+  //   load users.json
```

Comment:
nie ma potrzeby wrzucac takich komentarzy. jesli uwazasz ze jest tego za duzo to wydziel metode loadUsers